prdoc/pr_4902.prdoc

```diff
@@ -0,0 +1,14 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Removed `pallet::getter` usage from the pallet-vesting
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      This PR removed `pallet::getter`s from `pallet-vesting`s storage items.
+      When accessed inside the pallet, use the syntax `StorageItem::<T>::get()`.
+
+crates:
+  - name: pallet-vesting
+    bump: minor
```

Comment:
```suggestion
    bump: patch
```
Can try if the CI goes green with patch.

Reply:
we can mention `minor` according to your suggestion here: https://github.com/paritytech/polkadot-sdk/pull/4912#discussion_r1664528514